scripts/dockerrun.sh

```diff
@@ -119,20 +119,14 @@ docker run $INTERACTIVE -t --rm --sig-proxy=true \
     --name $DOTNET_BUILD_CONTAINER_NAME \
     -v $DOCKER_HOST_SHARE_DIR:/opt/code \
     -e CHANNEL \
-    -e CONNECTION_STRING \
```

Comment:
Why is it we don't need these anymore? In the very least, I would expect we need the access token.

Reply:
The access token is passed explicitly as a msbuild 'secret' parameter to whatever step needs it - my understanding is that if it is passed explicitly; defining it again in the environment is redundant.

Reply:
But without this, it won't flow into the docker container. Which is why all of these are here. When the build with docker, the env variables are set in the hosting machine, than we start a docker VM where the actual build happens. Not everything is passed down, only what is defined here.

Reply:
Reverting...